vmm/src/config.rs

```diff
@@ -635,10 +632,6 @@ impl DiskConfig {
             warn!("poll_queue parameter currently only has effect when used vhost_user=true");
         }
```

Comment:
Don't we want to tackle these warnings as part of the `validate()` too?

Reply:
Sadly it's not possible to do that as they have default values that get filled in for the API. In particularly these two are not testing if the values are set but whether they have been specified.

We could move to using `Option<>` types for these fields but I hope that we will get `EVENT_IDX` and WCE control in virtio-block soon enough (or that we default to vhost-user anyway.

Reply:
Yes that's fine, as long as we have the support coming in virtio, we'll be able to get rid of those warnings.